docs/recipes/Internationalizing-Your-Module.md

```diff
@@ -2,6 +2,250 @@
 
 # Internationalizing Your Module
 
-Recipe is forthcoming.
+Internationalization with [react-intl](https://github.com/formatjs/react-intl) is quick and simple in One App.
+
+*If you used the [One App Module Generator](https://github.com/americanexpress/one-app-cli/tree/master/packages/generator-one-app-module) to create the module some of these steps may already be completed.*
```

Comment:
can we bold this somehow, I cannot imagine anyone not using the generator and this tutorial adds a lot of boilerplate steps, which is good for explanation but not if I miss this note.

Reply:
and maybe an additional, we highly recommend using the generator and using these steps to explain each part

Reply:
@jsmartfo Updated, take a look.